src/app/components/ElectronicDeliveryForm/ElectronicDeliveryForm.jsx

```diff
@@ -37,6 +39,25 @@ class ElectronicDeliveryForm extends React.Component {
     this.handleUpdate = this.handleUpdate.bind(this);
   }
 
+  componentDidMount() {
+    const formState = fetchFromLocal('formstate');
```

Comment:
will fetchfromlocal throw an exception if formstate is not defined on the local state?